website/content/docs/intro/usecases/what-is-a-service-mesh.mdx

```diff
@@ -0,0 +1,112 @@
+
+Implementing dynamic network behavior is critical as more and more applications are deployed across different cloud providers (multi-cloud) and private data centers.
+Organizations may need to route network traffic to other infrastructure environments. Ensuring this traffic is secure is on top of mind for all organizations.
+Service meshes offer the ability to enforce network traffic encryption (mTLS) and authentication between all services. The service mesh can automatically generate an SSL certificate for each service and its instances.
+The certificate authenticates with other services inside the mesh and encrypts the TCP/UDP/gRPC connection with SSL.
+
+Fine-grained policies that dictate what services are allowed to communicate with each other is another benefit of a service mesh.
+Traditionally, services are permitted to communicate with other services through firewall rules.
+The traditional firewall (IP-based) model is difficult to enforce with dynamic infrastructure resources with a short lifecycle and frequently recycling IP addresses.
+As a result, network administrators have to open up network ranges to permit network traffic between services without differentiating the services generating the network traffic. However, a service mesh allows operators and developers to shift away from an IP-based model and focus more on service to service permissions.
+An operator defines a policy that only allows _service A_ to communicate with _service B_. Otherwise, the default action is to deny the traffic.
+This shift from an IP address-based security model to a service-focused model reduces the overhead of securing network traffic and allows an organization to take advantage of multi-cloud environments without sacrificing security due to complexity.
+
+## How Do You Implement a Service Mesh?
+
+Service meshes are commonly installed in Kubernetes clusters. There are also platform-agnostic service meshes available for non-Kubernetes-based workloads.
+For Kubernetes, most service mesh can be installed by operators through a [Helm chart](https://helm.sh/). Additionally, the service mesh may offer a CLI tool that supports the installation and maintenance of the service mesh.
+Non-Kubernetes based service meshes can be installed through infrastructure as code (IaC) products such as [Terraform](https://www.terraform.io/), CloudFormation, ARM Templates, Puppet, Chef, etc.
+
+## What is a Multi Platform Service Mesh?
+
+A multi-platform service mesh is capable of supporting various infrastructure environments. This can range from having the service mesh support Kubernetes and non-Kubernetes workloads, to having a service mesh span across various cloud environments (multi-cloud).
+
+## What is Consul?
+
+Consul is a multi-networking tool that offers a fully-featured service mesh solution that solves the networking and security challenges of operating microservices and cloud infrastructure.
```

Comment:
What is "multi-networking"? Feels like an undefined buzzword.

Reply:
I feel like we're missing something about heterogenous environments in here (e.g., multi-cloud, hybrid-cloud ... whatever combo you have)

Reply:
@jkirschner-hashicorp  the term "multi-networking" is something Neena has been wanting us to use. I would check with her, but you're right. I don't think there is a clear definition of the term. 

I'm adding multi-cloud, hybrid-cloud